tldextract/tldextract.py

```diff
@@ -281,7 +281,7 @@ def _add_extra_suffixes(self, suffixes):
             return frozenset(suffixes)
         return suffixes
 
-TLD_EXTRACTOR = TLDExtract()
+TLD_EXTRACTOR = TLDExtract(include_psl_private_domains=True)
```

Comment:
I like where this is going. I agree we must persist all public/private domains, but only include private results if the user opts into them. However this edit will include private domains for anybody who uses the export `extract`, just a few lines down. That would undo #19.